youths/schema/mutations.py

```diff
@@ -246,43 +246,45 @@ class UpdateYouthProfileInput(YouthProfileFields):
 class UpdateYouthProfileMutation(relay.ClientIDMutation):
     class Input:
         id = graphene.Argument(graphene.ID, required=True)
-        profile = UpdateYouthProfileInput(required=True)
+        youth_profile = UpdateYouthProfileInput(required=True)
+
+    youth_profile = graphene.Field(YouthProfileNode)
 
     @classmethod
     @staff_required
     @transaction.atomic
     def mutate_and_get_payload(cls, root, info, **input):
-        input_data = input.get("profile")
+        input_data = input.get("youth_profile")
 
         youth_profile = YouthProfile.objects.get(pk=from_global_id(input.get("id"))[1])
         youth_profile = update_youth_profile(
             input_data, youth_profile, manage_permission=True
         )
-        return UpdateMyYouthProfileMutation(profile=youth_profile)
+        return UpdateMyYouthProfileMutation(youth_profile=youth_profile)
 
 
 class UpdateMyYouthProfileMutation(relay.ClientIDMutation):
     class Input:
-        profile = UpdateYouthProfileInput(required=True)
+        youth_profile = UpdateYouthProfileInput(required=True)
```

Comment:
I'm not sure if it would make sense to have the input resemble the name of the mutation, but I decided to rename any `*MyYouthProfileInput`s to *`*YouthProfileInput`s. That felt logical.

Reply:
I concur. That naming does fit both cases.